util/metrics/metrics.go

```diff
@@ -0,0 +1,69 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// RegisterPrometheusMetrics register all prometheus metrics with the global
+// metrics handler.
+func RegisterPrometheusMetrics() {
+	prometheus.Register(BlockImportTimeHistogramSeconds)
+	prometheus.Register(CumulativeImportTimeCounter)
+	prometheus.Register(CumulativeTransactionsCounter)
+	prometheus.Register(ImportedTransactionsPerBlockHistogram)
+	prometheus.Register(CurrentRoundGauge)
+}
+
+const (
+
+	// ImportTimePerBlockHistogramName metric name.
```

Comment:
I think, if you put a comment right above `const (`, the linter will not require all these silly comments.